packages/manager/tools/sao-ovh-manager-module/template/src/translations/Messages_de_DE.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
This file shouldn't be removed. It is used to generate modules

Reply:
i've reset tools folder